.github/workflows/cd.yml

```
@@ -0,0 +1,154 @@
+
+      - name: Bump version
+        run: hatch bump ${{ needs.version-type.outputs.type }}
+
+      - name: Get bumped version
+        id: get-version
+        run: |
+          VERSION=$(hatch version)
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+
+      - name: Login to Azure - CI Subscription
+        uses: Azure/login@e15b166166a8746d1a47596803bd8c1b595455cf # v1.6.0
+        with:
+          creds: ${{ secrets.AZURE_KV_CI_SERVICE_PRINCIPAL }}
+
+      - name: Retrieve secrets
+        id: retrieve-secrets
+        uses: bitwarden/gh-actions/get-keyvault-secrets@main
+        with:
+          keyvault: "bitwarden-ci"
+          secrets: "github-gpg-private-key,
+            github-gpg-private-key-passphrase,
+            github-pat-bitwarden-devops-bot-repo-scope"
+
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@01dd5d3ca463c7f10f7f4f7b4f177225ac661ee4 # v6.1.0
+        with:
+          gpg_private_key: ${{ steps.retrieve-secrets.outputs.github-gpg-private-key }}
+          passphrase: ${{ steps.retrieve-secrets.outputs.github-gpg-private-key-passphrase }}
+          git_user_signingkey: true
+          git_commit_gpgsign: true
+
+      - name: Setup git
+        run: |
+          git config --local user.email "106330231+bitwarden-devops-bot@users.noreply.github.com"
+          git config --local user.name "bitwarden-devops-bot"
+
+      - name: Commit version bump
+        env:
+          OLD_VERSION: ${{ env.OLD_VERSION }}
+          VERSION: ${{ steps.get-version.outputs.version }}
+        if: ${{ github.event_name == 'push' }} || ${{ inputs.release_type != 'Dry Run' }}
+        run: |
+          git commit -am "Bump version from $OLD_VERSION to $VERSION"
+          git tag $VERSION
+          git push
+          git push --tags
+
+  release:
+    name: GitHub release
+    runs-on: ubuntu-22.04
+    needs: version-bump
+    if: ${{ github.event_name == 'push' }} || ${{ inputs.release_type != 'Dry Run' }}
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Create GitHub release
+        uses: ncipollo/release-action@2c591bcc8ecdcd2db72b97d6147f871fcd833ba5 # v1.14.0
+        with:
+          commit: ${{ github.sha }}
+          tag: v${{ steps.version-bump.outputs.version }}
+          name: Version version-bump
+          token: ${{ secrets.GITHUB_TOKEN }}
+          draft: false
+
+  deploy:
+    name: Deploy workflow-linter (v2)
+    runs-on: ubuntu-22.04
+    needs: version-bump
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Set up Python
+        uses: actions/setup-python@0a5c61591373683505ea898e09a3ea4f39ef2b9c # v5.0.0
+        with:
+          python-version-file: ".python-version"
+
+      - name: Install hatch
+        run: pip install hatch
+
+      - name: Login to Azure
```

⛏️  Consistent with other uses from above.
```suggestion
      - name: Login to Azure - CI Subscription
```